apps/console/src/features/core/configs/app.ts

```diff
@@ -41,6 +42,7 @@ import { I18nConstants } from "../constants";
 import { DeploymentConfigInterface, ServiceResourceEndpointsInterface, UIConfigInterface } from "../models";
 import { store } from "../store";
 
+
```

Comment:
Shall we remove this extra new line?

Reply:
fixed with https://github.com/wso2/identity-apps/pull/3786/commits/dd8a9366531a26a1a44411371fe1db17b81925ae